ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/execution/StatementExecutor.java

```diff
@@ -35,12 +36,14 @@
    * @param sessionProperties the session properties
    * @param executionContext the context in which to execute it
    * @param serviceContext the services to use to execute it
+   * @param restConfig the rest config allows to plug-in custom behavior, e.g. error handling
```

Comment:
Assuming we move the new config from KsqlRestConfig into KsqlConfig instead, then we don't need this new argument. KsqlConfig is already captured as part of ConfiguredStatement (the first param).

Reply:
Good idea! Done in 4d4ea68.